site/docs/concepts/vault_plugins.md

```diff
@@ -7,14 +7,20 @@ This project uses secrets plugins to retrieve dataset credentials by the running
 
 Additional secret plugins can be developed to retrieve credentials additional location. This [tutorial](https://learn.hashicorp.com/tutorials/vault/plugin-backends?in=vault/app-integration) can serve as a good starting point to learn about Vault plugin development.
 
-The following steps are for configuring a secret plug-in for Fybrik:
+## Before you begin
 
-1. Enable the plugin during Vault server initialization in a specific path. 
-<br/>An example of that can be found in helm chart [values.yaml](https://github.com/fybrik/fybrik/blob/master/third_party/vault/vault-single-cluster/values.yaml) file in the project where [Vault-plugin-secrets-kubernetes-reader](https://github.com/fybrik/vault-plugin-secrets-kubernetes-reader) plugin is enabled in `kubernetes-secrets` path:
+Ensure that you have the [Vault v1.9.x](https://www.vaultproject.io/docs/commands) to execute Vault CLI commands.
```

Comment:
I don't think this is pointing to the right location - shouldn't the reference be:
https://www.vaultproject.io/downloads

Reply:
Fixed. Thanks